File: src/fiefapp.svelte/src/models/settings/selection-alternatives.ts
import type { Inheritance } from '../inheritance';
import type { Livingcondition } from '../livingcondition';
import type { Road } from '../road';

export interface ISelectionAlternatives {
  roads: Road[];
  inheritances: Inheritance[];
  livingconditions: Livingcondition[];
}

export const initialSelectionAlternatives: ISelectionAlternatives = {
  roads: [],
  inheritances: [],
  livingconditions: [],
};

File: src/fiefapp.entities/SelectionAlternatives.cs
using System.Collections.Generic;

namespace fiefapp.entities
{
    public class SelectionAlternatives
    {
        public List<Inheritance> Inheritances { get; set; }
        public List<Livingcondition> Livingconditions { get; set; }
        public List<Road> Roads { get; set; }
    }
}